aten/src/ATen/native/mkldnn/Linear.cpp

```diff
@@ -59,6 +61,16 @@ namespace at::native {
 Tensor mkldnn_linear(
     const Tensor& self,
     const Tensor& weight_t, const std::optional<Tensor>& bias_opt) {
+#if defined(__aarch64__) && AT_MKLDNN_ACL_ENABLED()
```

Comment:
ditto: explain why we implement specific path for aarch64.

Reply:
Done